Mods/ModVarietyPack/Defs/ThingDefs/Plants_Cultivated.xml

```diff
@@ -61,8 +61,8 @@
       <soundHarvesting>Harvest_Standard</soundHarvesting>
       <soundHarvestFinish>Harvest_Standard_Finish</soundHarvestFinish>
       <sowWork>150</sowWork>
-      <!--<fertilityMin>0.5</fertilityMin>
-      <fertilityFactorGrowthRate>1.0</fertilityFactorGrowthRate>-->
+      <fertilityMin>0.5</fertilityMin>
+      <fertilityFactorGrowthRate>1.0</fertilityFactorGrowthRate>
```

Comment:
This won't work. 
I have them defined for each plant seperate, See the top of this file.
If the factor is 1, the boost would be too big. 
Example: If you have a plant that needs at least 2.0 fertility to grow, it would then grow with 200% by default.